spyder/plugins/variableexplorer/widgets/objectexplorer/utils.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2016 Pepijn Kenter.
+# Copyright (c) 2019- Spyder Project Contributors
+#
+# Components of objectbrowser originally distributed under
+# the MIT (Expat) license.
+# Licensed under the terms of the MIT License; see NOTICE.txt in the Spyder
+# root directory for details
+# -----------------------------------------------------------------------------
+
+# Standard library imports
+import logging
+
+
+def logging_basic_config(level='INFO'):
```

Comment:
This function is not necessary because we already setup our logging in a particular way. Besides, it doesn't seem to be used in the rest of the code in this PR, so please remove it.